app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
```

Comment:
- [ ] Kindly add corresponding view file for the  index controller actions, you currently have only the show view file for your posts controller 👍🏿